docs/docs/populate-connections-views.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Populate Connections and Views
+layout: docs
+---
+
+When you get a workbook with the TSC library, the response from Tableau Server does not include information about the
+views or connections that make up the workbook. Similarly, when you get a data source, the response does not include
+information about the connections that make up the data source. This is a result of the design of the Tableau Server
+REST API, which optimizes the size of responses by only returning what you ask for explicitly.
```

Comment:
Do we really need to explain why? It's not just optimization it's also"blah blah REST and resources and collections and blah blah"